### PR TITLE
Chatbot user manual

### PR DESCRIPTION
**Pull request by ShellyXueHan**
### Updates:
This is the user manual for the community chatbot.

Please provide your feedback about the user manual below as comments!

### CODE REVIEW

File: src/docs/training-and-learning/rocky-guide.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+### Here’s how I can help you out:
+- **Got Questions about OpenShift?** 🤔 I’ve got answers! Just ask me anything related to OpenShift platform usage, and I’ll fetch the info from our knowledge base faster than you can say “Kubernetes!”
+- **Need Info from the websites above?** 🌐 I’m a pro at digging up details from these sites. Whether it’s a questions for the how-tos, troubleshooting, or best practices, as long as it's there somewhere, I’ve got you covered.
+
+> Remember, I’m all about OpenShift and those specific websites. If you ask me something outside my wheelhouse, I’ll redirect you to our community powered channel at #devops-how-to for some real human interaction who can better help you out. 
+
+- **Got Feedback?** 💬 Yes please! I need feedback to continue to grow. Simply give it a thumb up or down to my answer and I'll take it from there! If you have any feedback or suggestions more than that, please ping my human care giver by `@shelly.han`. We’d love to hear from you!
+
+So, let’s get started! Head over to the [rocky-chat channel](https://chat.developer.gov.bc.ca/group/rocky-chat) and call me with `!Rocky` followed by your questions. Ask away!
```
Comment (StevenBarre) [inline src/docs/training-and-learning/rocky-guide.md:17]:
```suggestion
So, let’s get started! Head over to the [rocky-chat channel](https://chat.developer.gov.bc.ca/channel/rocky-chat) and call me with `!Rocky` followed by your questions. Ask away! 
```
It's a channel not a group.

File: src/docs/training-and-learning/rocky-guide.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Platform Services GenAI Chatbot - Rocky
+
+🎉 Hey there! I’m Rocky from the Platform Services Team, your friendly AI assistant in RocketChat! 🤖✨ I focus on the OpenShift platform and the shared services related to it. I've been equipped with all the knowledge from the following sources:
+- [BC Government Private Cloud Private Cloud website](http://digital.gov.bc.ca/cloud/services/private)
```
Comment (Pilargit12) [inline src/docs/training-and-learning/rocky-guide.md:4]:
```suggestion
- [B.C. Government Private Cloud Private Cloud website](http://digital.gov.bc.ca/cloud/services/private)
```

File: src/docs/training-and-learning/rocky-guide.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+- [Technical Documentation website](https://developer.gov.bc.ca/docs/default/component/platform-developer-docs/)
```
Comment (Pilargit12) [inline src/docs/training-and-learning/rocky-guide.md:5]:
```suggestion
- [Private cloud technical documentation website](https://developer.gov.bc.ca/docs/default/component/platform-developer-docs/)
```

File: src/docs/training-and-learning/rocky-guide.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+- [BCGov Internal StackOverflow](https://stackoverflow.developer.gov.bc.ca/)
+
+
+### Here’s how I can help you out:
+
+- **Got Questions about OpenShift?** 🤔 I’ve got answers! Just ask me anything related to OpenShift platform usage, and I’ll fetch the info from our knowledge base faster than you can say “Kubernetes!”
```
Comment (Pilargit12) [inline src/docs/training-and-learning/rocky-guide.md:11]:
```suggestion
- **Have questions about OpenShift?** 🤔 I’ve got answers! Ask me anything about using the OpenShift platform, and I’ll pull info from our knowledge base faster than you can say “Kubernetes!”
```

File: src/docs/training-and-learning/rocky-guide.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+- **Need Info from the websites above?** 🌐 I’m a pro at digging up details from these sites. Whether it’s a questions for the how-tos, troubleshooting, or best practices, as long as it's there somewhere, I’ve got you covered.
```
Comment (Pilargit12) [inline src/docs/training-and-learning/rocky-guide.md:12]:
```suggestion
- **Need info from the sites above?** 🌐 I’m great at finding details! Whether it’s how-tos, troubleshooting tips, or best practices, if it’s on those sites, I’ll track it down for you.
```

File: src/docs/training-and-learning/rocky-guide.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+> Remember, I’m all about OpenShift and those specific websites. If you ask me something outside my wheelhouse, I’ll redirect you to our community powered channel at #devops-how-to for some real human interaction who can better help you out.
```
Comment (Pilargit12) [inline src/docs/training-and-learning/rocky-guide.md:14]:
```suggestion
> Remember, I’m here to help with OpenShift and the websites I know. If your question is outside my expertise, I’ll point you to our community-powered channel, [#devops-how-to](https://chat.developer.gov.bc.ca/channel/devops-how-to), where real humans can assist you better.
```

File: src/docs/training-and-learning/rocky-guide.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+
+### Here’s how I can help you out:
+
+- **Got Questions about OpenShift?** 🤔 I’ve got answers! Just ask me anything related to OpenShift platform usage, and I’ll fetch the info from our knowledge base faster than you can say “Kubernetes!”
+- **Need Info from the websites above?** 🌐 I’m a pro at digging up details from these sites. Whether it’s a questions for the how-tos, troubleshooting, or best practices, as long as it's there somewhere, I’ve got you covered.
+
+> Remember, I’m all about OpenShift and those specific websites. If you ask me something outside my wheelhouse, I’ll redirect you to our community powered channel at #devops-how-to for some real human interaction who can better help you out. 
+
+- **Got Feedback?** 💬 Yes please! I need feedback to continue to grow. Simply give it a thumb up or down to my answer and I'll take it from there! If you have any feedback or suggestions more than that, please ping my human care giver by `@shelly.han`. We’d love to hear from you!
```
Comment (Pilargit12) [inline src/docs/training-and-learning/rocky-guide.md:16]:
```suggestion
- **Got feedback?** 💬 I’d love it! Your feedback helps me improve. Give my answer a thumbs up or down, and I’ll take it from there. If you have more detailed feedback or suggestions, please ping my human caregiver at `@shelly.han`. We’re eager to hear from you! 
```

File: src/docs/training-and-learning/rocky-guide.md [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+
+So, let’s get started! Head over to the [rocky-chat channel](https://chat.developer.gov.bc.ca/channel/rocky-chat) and call me with `!rocky`, followed by your questions. Ask away!
```
Comment (Pilargit12) [inline src/docs/training-and-learning/rocky-guide.md:18]:
```suggestion
Let’s get started! Head over to the [rocky-chat channel](https://chat.developer.gov.bc.ca/channel/rocky-chat) and call me with `!rocky`, followed by your questions. Ask away!
```

File: src/docs/training-and-learning/rocky-guide.md [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+
+Note that while interacting with me, please do not include any sensitive personal information in your conversation. This would include any reference to religious beliefs, government service history, sexuality and gender details, health info, private ID numbers, etc.
```
Comment (Pilargit12) [inline src/docs/training-and-learning/rocky-guide.md:20]:
```suggestion
**Important**: Please don’t include sensitive personal information when chatting with me. This includes details about religious beliefs, government service history, sexuality, gender, health, private ID numbers, or similar data.
```

File: src/docs/training-and-learning/rocky-guide.md [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+
+### Wonder how I work?
```
Comment (Pilargit12) [inline src/docs/training-and-learning/rocky-guide.md:22]:
```suggestion
### Curious about how I work?
```

File: src/docs/training-and-learning/rocky-guide.md [hunk 11]
```diff
@@ -0,0 +1,27 @@
...
+
+There are several parts of me that work together:
```
Comment (Pilargit12) [inline src/docs/training-and-learning/rocky-guide.md:24]:
```suggestion
Here’s a quick look behind the scenes:
```

File: src/docs/training-and-learning/rocky-guide.md [hunk 12]
```diff
@@ -0,0 +1,27 @@
...
+- My DNA sequence 🧬 is recorded in Rocky repo: https://github.com/bcgov/rockychat
```
Comment (Pilargit12) [inline src/docs/training-and-learning/rocky-guide.md:25]:
```suggestion
- My DNA 🧬 is stored in the [Rocky repo](https://github.com/bcgov/rockychat)
```

File: src/docs/training-and-learning/rocky-guide.md [hunk 13]
```diff
@@ -0,0 +1,27 @@
...
+- I live in the Silver cluster, as you might have guessed, I have a sibling called Rocky Test. I'm jealous of him, he gets to try out everything new before me! But anyway, only I get to talk to you 😃
```
Comment (Pilargit12) [inline src/docs/training-and-learning/rocky-guide.md:26]:
```suggestion
- I live in the Silver cluster alongside my sibling, Rocky Test. (He gets to test new features first—I’m totally not jealous! 😅) But I’m the one you get to talk to!
```

File: src/docs/training-and-learning/rocky-guide.md [hunk 14]
```diff
@@ -0,0 +1,27 @@
...
+- My brain 🧠 is always evolving, currently you are talking to GPT4 from [Azure OpenAI](https://azure.microsoft.com/en-us/products/ai-services/openai-service). I'm always learning and improving!
```
Comment (Pilargit12) [inline src/docs/training-and-learning/rocky-guide.md:27]:
```suggestion
- My brain 🧠 is powered by GPT-4 through [Azure OpenAI](https://azure.microsoft.com/en-us/products/ai-services/openai-service) and I’m always learning and improving!
```

File: src/docs/training-and-learning/rocky-guide.md [hunk 15]
```diff
@@ -0,0 +1,27 @@
+# Platform Services GenAI Chatbot - Rocky
```
Comment (Pilargit12) [inline src/docs/training-and-learning/rocky-guide.md:1]:
```suggestion
# Platform Services GenAI chatbot: Rocky
```

**Review by Pilargit12 — CHANGES_REQUESTED** [review]
Hi @ShellyXueHan, went through the document, looks great! Just a couple of things for clarity and plain language. 

I was wondering if you could add perhaps something about support if something is broken? Who are users supposed to contact?

**Review by Pilargit12 — APPROVED** [review]

File: src/docs/training-and-learning/rocky-guide.md [hunk 16]
```diff
@@ -0,0 +1,32 @@
+# Platform Services GenAI chatbot: Rocky
+
+🎉 Hi! I’m Rocky from the Platform Services team, your friendly AI assistant in Rocket.Chat! 🤖✨ I’m here to help with the OpenShift platform and its related shared services. I’ve been trained using knowledge from these sources:
```
Comment (Pilargit12) [inline src/docs/training-and-learning/rocky-guide.md:3]:
```suggestion
Hi! I’m Rocky from the Platform Services team, your friendly AI assistant in Rocket.Chat! 🤖✨ I’m here to help with the OpenShift platform and its related shared services. I’ve been trained using knowledge from these sources:
```

File: src/docs/training-and-learning/rocky-guide.md [hunk 17]
```diff
@@ -0,0 +1,32 @@
...
+**Important**: Please don’t include sensitive personal information when chatting with me. This includes details about religious beliefs, government service history, sexuality, gender, health, private ID numbers, or similar data.
+
+## Curious about how I work?
+
+Here’s a quick look behind the scenes:
+- My DNA 🧬 is stored in the [Rocky repo](https://github.com/bcgov/rockychat)
+- I live in the Silver cluster alongside my sibling, Rocky Test. (He gets to test new features first—I’m totally not jealous! 😅) But I’m the one you get to talk to!
+- My brain 🧠 is powered by GPT-4 through [Azure OpenAI](https://azure.microsoft.com/en-us/products/ai-services/openai-service) and I’m always learning and improving!
+
+## If something went wrong:
```
Comment (Pilargit12) [inline src/docs/training-and-learning/rocky-guide.md:29]:
```suggestion
## Troubleshooting Rocky chatbot
```

File: src/docs/training-and-learning/rocky-guide.md [hunk 18]
```diff
@@ -0,0 +1,32 @@
+# Platform Services GenAI chatbot: Rocky
+
+🎉 Hi! I’m Rocky from the Platform Services team, your friendly AI assistant in Rocket.Chat! 🤖✨ I’m here to help with the OpenShift platform and its related shared services. I’ve been trained using knowledge from these sources:
+- [B.C. Government Private Cloud Private Cloud website](http://digital.gov.bc.ca/cloud/services/private)
+- [Private cloud technical documentation website](https://developer.gov.bc.ca/docs/default/component/platform-developer-docs/)
+- [BCGov Internal StackOverflow](https://stackoverflow.developer.gov.bc.ca/)
+
+
+## Here’s how I can help you out:
```
Comment (Pilargit12) [inline src/docs/training-and-learning/rocky-guide.md:9]:
```suggestion
## How I help you
```

File: src/docs/training-and-learning/rocky-guide.md [hunk 19]
```diff
@@ -0,0 +1,32 @@
...
+## Curious about how I work?
+
+Here’s a quick look behind the scenes:
+- My DNA 🧬 is stored in the [Rocky repo](https://github.com/bcgov/rockychat)
+- I live in the Silver cluster alongside my sibling, Rocky Test. (He gets to test new features first—I’m totally not jealous! 😅) But I’m the one you get to talk to!
+- My brain 🧠 is powered by GPT-4 through [Azure OpenAI](https://azure.microsoft.com/en-us/products/ai-services/openai-service) and I’m always learning and improving!
+
+## If something went wrong:
+
+- If you asked a question and didn't hear back from me, please first check if you tagged me with `!rocky` (I'm not case sensitive but don't miss the exclamation mark). If still no response, please tag `@shelly.han` for support!
```
Comment (Pilargit12) [inline src/docs/training-and-learning/rocky-guide.md:31]:
```suggestion
- If you asked a question and didn't hear back from me, please first check if you tagged me with `!rocky` (I'm not case sensitive but don't miss the exclamation mark). If there's no response still, please tag `@shelly.han` for support!
```

File: src/docs/training-and-learning/rocky-guide.md [hunk 20]
```diff
@@ -0,0 +1,32 @@
...
+- If there are specific issues, concerns, and feature requests, please create them as a Github Issue in the [Rocky repo](https://github.com/bcgov/rockychat).
```
Comment (Pilargit12) [inline src/docs/training-and-learning/rocky-guide.md:32]:
```suggestion
- If there are specific issues, concerns and/or feature requests, please create them as a Github Issue in the [Rocky repo](https://github.com/bcgov/rockychat).
```

**Review by Pilargit12 — COMMENTED** [review]
Just minor updates and then should be good to go!

File: mkdocs.yml [hunk 21]
```diff
@@ -82,6 +82,7 @@ nav:
 - Training and learning:
   - Training from the Platform Services Team: docs/training-and-learning/training-from-the-platform-services-team.md
   - External training resources: docs/training-and-learning/training-external-resources.md
+  - Platform Services GenAI chatbot: Rocky: docs/training-and-learning/rocky-guide.md
```
Comment (ShellyXueHan) [inline mkdocs.yml:85]:
```suggestion
  - Platform Services GenAI chatbot Rocky: docs/training-and-learning/rocky-guide.md
```

**Pull request merged**